ecpy_hqc_legacy/tasks/tasks/instr/views/run_awg_views.enaml

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright 2015-2016 by EcpyHqcLegacy Authors, see AUTHORS for more details.
+#
+# Distributed under the terms of the BSD license.
+#
+# The full license is in the file LICENCE, distributed with this software.
+# -----------------------------------------------------------------------------
+"""Views for the microwave sources related tasks.
+
+"""
+from __future__ import (division, unicode_literals, print_function,
+                        absolute_import)
+
+from enaml.core.api import Conditional
+from enaml.widgets.api import (GroupBox, Label, Field, ObjectCombo, CheckBox)
+from enaml.stdlib.fields import FloatField
+from enaml.layout.api import factory
+
+from textwrap import fill
```

Comment:
stdlib imports should be before external libraries imports

Reply:
I don't understand, is this ok? (see edit)